test/functional/search_controller_test.rb

```diff
@@ -77,4 +77,19 @@ class SearchControllerTest < ActionController::TestCase
     assert_equal nodes_with_trawl, nodes_with_trawls
     assert_response :success
   end
+
+  test "search for purple-air also includes results for purpleair and vice versa" do
```

Comment:
Let's also add "for hyphenated searches" just so it's easy to find the word `hyphen` when searching the codebase!